src/Uno.UWP/Devices/Midi/MidiOutPort.iOSmacOS.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading.Tasks;
+using Uno.Devices.Enumeration.Internal;
+using Uno.Devices.Enumeration.Internal.Providers.Midi;
+using Windows.Devices.Enumeration;
+using Windows.Foundation;
+using CoreMidi;
+using MidiInfo = CoreMidi.Midi;
+
+namespace Windows.Devices.Midi
+{
+	public partial class MidiOutPort : IDisposable
+	{
+		private MidiOutPort()
+		{
+			
+		}
+
+		public string DeviceId { get; private set; }
+
+		public static IAsyncOperation<IMidiOutPort> FromIdAsync(string deviceId) =>
+			FromIdInternalAsync(deviceId).AsAsyncOperation();
+
+
+		public void Dispose()
+		{
+		}
+
+		private static async Task<IMidiOutPort> FromIdInternalAsync(string deviceId)
+		{
+			//var parsedIdentifier = DeviceInformation.ParseDeviceId(deviceId);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)